src/loaders/common/loader_helper.py

```diff
@@ -114,8 +115,12 @@ def process_columnar_meta(
     spec = load_spec(product_id, kbase_collection)
     columns = list()
     for col_spec in spec.columns:
-        key, col_type = col_spec.key, col_spec.type
-        values = _convert_values_to_type(docs, key, col_type, ignore_missing=ignore_missing)
+        key, col_type, no_cast = col_spec.key, col_spec.type, col_spec.no_cast
```

Comment:
I would just access these from col_type personally

Reply:
you mean `col_spec`? like pass `col_spec` to the `_convert_values_to_type`?

Reply:
No, just get rid of that line and do 
```python
        values = _convert_values_to_type(docs,
                                         col_spec.key,
                                         col_spec.col_type,
                                         ignore_missing=ignore_missing,
                                         no_cast=col_spec.no_cast)
```

and same for the `col_type` references below

Reply:
👍